src/client/components/EventsPageComplete/EventsPageComplete.js

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+// import EventsPage from '../EventsPage/EventsPage.component';
+import { AppHeader } from '../Appheader/AppHeader.component';
+import Footer from '../footer/footer';
+import './EventsPageComplete.styles.css';
+import Header from '../Heading/Heading';
+import { CardWithEventsOrStudyGroups } from '../CardWithEventsOrStudyGroups/CardWithEventsOrStudyGroups';
+
+function EventsPageComplete() {
```

Comment:
Why do you have this `Complete` version of a page while also having the `EventsPage` component?